### DISCUSSION TITLE
I want a reviewer

### DISCUSSION THREAD

**Discussion by scratchusernamemrtbts** [ScratchAddons/ScratchAddons#1669] [Translation] [answered]
I am translating in to thai but I don’t have any reviewer to help me review my work

**Comment by retronbv** [comment]
Does your family or anybody you know thai? They could help you review

**Comment by TheColaber** [comment, marked as answer]
People will review when they can. Ask your family like @retronbv said... please do not ask people on gh discussions.

**Reply by scratchusernamemrtbts** [reply]
ok. i will ask @eddyting

**Reply by eddyting** [reply]
what do i have to review?

**Reply by scratchusernamemrtbts** [reply]
@eddyting join transifex
Then tell you transifex username

**Reply by scratchusernamemrtbts** [reply]
@eddyting i thought you wren’t active

**Reply by scratchusernamemrtbts** [reply]
@eddyting go to https://www.transifex.com/signup/ to create a transifex account
You can use GitHub then you don’t need to use your real name in your full name form
Then finish creating it
Then go to your profile setting and make it public
Then tell me your username

**Reply by eddyting** [reply]
i dont understand anything haha- if you want me to review a translation can you just send me the text?

**Reply by scratchusernamemrtbts** [reply]
That’s too hard because the text is very long
@eddyting  try creating a transifex account first at https://www.transifex.com/signup/

**Reply by eddyting** [reply]
how do i join?

**Reply by scratchusernamemrtbts** [reply]
did you create an account yet?
if you did then make it public and tell me your username

**Reply by eddyting** [reply]
how do i tell my trasnfix name how do i join whatttt

**Reply by scratchusernamemrtbts** [reply]
บ อ ก อี เ ม ล ม า  เ ดี๋ ย ว ช่ ว ย

**Reply by scratchusernamemrtbts** [reply]
ใ           ห้     ตั้     งร   หั       ส  ว่       า อ       ะ ไ   ร

**Reply by scratchusernamemrtbts** [reply]
sorry i have to ask in thai for security reason
google translate can't translate it

**Reply by eddyting** [reply]
wasabianda23

**Reply by eddyting** [reply]
sorry i was gone for long

**Reply by eddyting** [reply]
okay, i join transifex now, its eddyting
